src/components/events/Event.js

```diff
@@ -1,7 +1,53 @@
+"use client";
 import React from "react";
-
-const Event = () => {
-  return <div>Event</div>;
+// style the component here
+const Event = ({ title, location, start, color, description, date }) => {
+  return (
+    <div className="flex flex-row font-urbanist">
+      <div
+        className={`rounded-l-lg text-center p-4 ${color} text-winc-white flex-col`}
+      >
+        <div className="text-5xl place-items-center font-semibold">{date}</div>
+        <div className="font-ultralight">{start}</div>
+      </div>
+      <div className="rounded-r-lg bg-winc-beige p-3 text-winc-black">
```

Comment:
you are using too many divs actually. Make sure you check the reference code. You can even use that as a template and then edit on top of it. Make sure to test your code.